reflex/utils/prerequisites.py

```diff
@@ -278,20 +278,36 @@ def initialize_app_directory(app_name: str, template: constants.Templates.Kind):
     )
 
 
+def get_project_hash_if_exists():
+    """Check if the reflex.json file exists in .web folder and 
+    if it does then extract the project_hash number so that we 
+    do not overwrite it every time we run the reflex init command"""
+    if os.path.exists(constants.Reflex.JSON):
+        # Open and read the file
+        with open(constants.Reflex.JSON, 'r') as file:
+            data = json.load(file)
+            project_hash = data["project_hash"]
+            return project_hash
+    else:
```

Comment:
Don't need the `else` since the `if` returns. Also generally may be cleaner to do the early return cases first:

```python
if not os.path.exists(...):
    return None
    
with open ...
```

Reply:
changed